src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1896,15 +1896,15 @@
   <!-- A subpattern, either in a positional or property part of a recursive pattern. -->
   <Node Name="BoundSubpattern" Base="BoundNode">
     <!-- The tuple element or parameter in a positional pattern, or the property or field in a property pattern. -->
-    <Field Name="Symbol" Type="Symbol" Null="allow"/>
+    <Field Name="Symbol" Type="Symbol?"/>
     <Field Name="Pattern" Type="BoundPattern" Null="disallow"/>
```

Comment:
> Null="disallow" [](start = 45, length = 16)

At some point, we could update the generator to recognize non-nullable types and enforce them (so we can remove `Null="disallow"`)

Reply:
Agreed